scripts/run-app.sh

```diff
@@ -1,55 +1,56 @@
 #!/usr/bin/env sh
 
 log() {
-  local message=$1
-  local iso_8601_timestamp=$(date +%Y-%m-%dT%H:%M:%S%z)
-  echo '{"@version": 1, "@timestamp": "'"$iso_8601_timestamp"'", "level": "INFO", "message": "'"$1"'"}'
+  message=$1
+  iso_8601_timestamp=$(date +%Y-%m-%dT%H:%M:%S%z)
+  echo '{"@version": 1, "@timestamp": "'"$iso_8601_timestamp"'", "level": "INFO", "message": "'"$message"'"}'
```

Comment:
What ensures message does not contain `"`? You might also want to use printf to avoid all the quoting.

Reply:
That… is very true. Good catch! Now to figure out how to escape the `"`s…